composer.json

```diff
@@ -28,15 +28,15 @@
     "require-dev": {
         "twig/twig": "~1.12|~2.0",
         "doctrine/orm": "~2.1",
-        "doctrine/phpcr-odm": "^1.3",
-        "jackalope/jackalope-doctrine-dbal": "^1.0",
+        "doctrine/phpcr-odm": "~1.3",
+        "jackalope/jackalope-doctrine-dbal": "~1.1.5",
         "propel/propel1": "~1.7",
-        "symfony/yaml": "2.*",
-        "symfony/translation": "~2.0",
-        "symfony/validator": "^2.2",
-        "symfony/form": "~2.1",
-        "symfony/filesystem": "2.*",
-        "phpunit/phpunit": "^4.0|^5.0"
+        "symfony/yaml": "2.*|~3.0",
+        "symfony/translation": "~2.0|~3.0",
+        "symfony/validator": "~2.0|3.0",
```

Comment:
Ah yes, I see they have since updated from when I first made these changes. I'll fix to match what's currently in master for low bounds.